src/Aspire.Hosting/ApplicationModel/ReferenceExpression.cs

```diff
@@ -67,8 +67,8 @@ private ReferenceExpression(string format, IValueProvider[] valueProviders, stri
     /// Gets the value of the expression. The final string value after evaluating the format string and its parameters.
     /// </summary>
     /// <param name="cancellationToken">A <see cref="CancellationToken"/>.</param>
-    /// <returns></returns>
-    public async ValueTask<string?> GetValueAsync(CancellationToken cancellationToken)
+    /// <param name="context">An optional <see cref="NetworkIdentifier"/> context for resolving the value.</param>
+    public async ValueTask<string?> GetValueAsync(CancellationToken cancellationToken, NetworkIdentifier? context)
```

Comment:
Why does this take a NetworkIdentifier? It should take the ValueProviderContext

Reply:
Yep, that is a miss, thanks. Fixed